tools/output-number-of-buffer-segments.rb

```diff
@@ -0,0 +1,49 @@
+#!/bin/ruby
+
+require "open3"
+require "json"
+require "date"
+
+if ARGV.size() == 0
+  puts "Usage: $0 DATABASE_PATH"
+  puts " e.g.: $0 ~/database/db"
+  exit
```

Comment:
```suggestion
  exit(false)
```

Reply:
ping

Reply:
Ah...I'm Sorry.
This was a correction omission.